### PR TITLE
Update netty for cve

### PR DESCRIPTION
**Pull request by siddvenk**
## Description ##

Brief description of what this PR is about

- If this change is a backward incompatible change, why must this change be made?
- Interesting edge cases to note here

### CODE REVIEW

File: serving/docker/scripts/install_djl_serving.sh [hunk 1]
```diff
@@ -27,3 +27,20 @@ else
...
+curl https://repo1.maven.org/maven2/io/netty/netty-codec-http/4.1.94.Final/netty-codec-http-4.1.94.Final.jar -f -o netty-codec-http-4.1.94.Final.jar
+curl https://repo1.maven.org/maven2/io/netty/netty-common/4.1.94.Final/netty-common-4.1.94.Final.jar -f -o netty-common-4.1.94.Final.jar
+curl https://repo1.maven.org/maven2/io/netty/netty-handler/4.1.94.Final/netty-handler-4.1.94.Final.jar -f -o netty-handler-4.1.94.Final.jar
+curl https://repo1.maven.org/maven2/io/netty/netty-resolver/4.1.94.Final/netty-resolver-4.1.94.Final.jar -f -o netty-resolver-4.1.94.Final.jar
+curl https://repo1.maven.org/maven2/io/netty/netty-transport/4.1.94.Final/netty-transport-4.1.94.Final.jar  -f -o netty-transport-4.1.94.Final.jar
+curl https://repo1.maven.org/maven2/io/netty/netty-transport-native-epoll/4.1.94.Final/netty-transport-native-epoll-4.1.94.Final-linux-x86_64.jar -f -o netty-transport-native-epoll-4.1.94.Final-linux-x86_64.jar
+curl https://repo1.maven.org/maven2/io/netty/netty-transport-native-kqueue/4.1.94.Final/netty-transport-native-kqueue-4.1.94.Final-osx-aarch_64.jar -f -o netty-transport-native-kqueue-4.1.94.Final-osx-aarch_64.jar
```
Comment (lanking520) [inline serving/docker/scripts/install_djl_serving.sh:44]: do we need to ship OSX jars?

Comment (siddvenk) [reply serving/docker/scripts/install_djl_serving.sh:44]: we don't need to, I can remove these. I wrote a script to translate the existing netty jars into the updated ones, so they're all here. 

Though, if anyone is still pulling these from dockerhub and trying to use python engine on mac, i guess that would no longer work.

Comment (frankfliu) [reply serving/docker/scripts/install_djl_serving.sh:44]: We don't need them, but the package is small

Comment (lanking520) [reply serving/docker/scripts/install_djl_serving.sh:44]: they will run in a linux container, OSX jar will not work

Comment (siddvenk) [reply serving/docker/scripts/install_djl_serving.sh]: I removed the osx jars

**Review by lanking520 — APPROVED** [review]

**Pull request merged**